frappe/public/scss/common/grid.scss

```diff
@@ -58,10 +58,17 @@
 }
 
 .form-grid .template-row {
-	width: calc(100% - 30px);
+	width: calc(100% - 20px);
 	padding: 8px 15px;
 }
 
+.template-row-index {
+	text-align: left;
+	margin-right: -50px;
+	padding: 8px 10px 0 15px;
+	width: 70px;
```

Comment:
How are we getting these `px` values? Can we use `calc` and CSS variables to get these values?